apps/devhub/utils.py

```diff
@@ -1,8 +1,21 @@
 from django.conf import settings
+from django.db.models import Q
+from django.forms import ValidationError
 
+from celery import chain, group
 from tower import ugettext as _
 
+import amo
+from addons.models import Addon
 from amo.utils import escape_all
+from files.models import File, FileUpload
+from files.utils import parse_addon
+from validator.version import Version
+from . import tasks
+
+
+# FIXME: Move to `validator.constants`.
+SEVERITIES = {'trivial', 'low', 'medium', 'high'}
```

Comment:
Do you want to import from https://github.com/mozilla/amo-validator/blob/1b3eee943dbe05c49da1de0a859619bf22c599da/validator/errorbundler.py#L34 instead? Is that what your comment means?